4.6/alpine/Dockerfile

```diff
@@ -0,0 +1,48 @@
+FROM alpine:3.4
+
+ENV NPM_CONFIG_LOGLEVEL info
+ENV NODE_VERSION 4.6.1
+
+RUN addgroup -S node \
+    && adduser -D -S -h /var/cache/node -s /sbin/nologin -G node node \
+    && apk add --no-cache \
+        libstdc++ \
+    && apk add --no-cache --virtual .build-deps \
+        binutils-gold \
+        curl \
+        g++ \
+        gcc \
+        gnupg \
+        libgcc \
+        linux-headers \
+        make \
+        paxctl \
```

Comment:
Sure? You should have this [kind of line if yes](https://github.com/rezzza/docker-node/blob/cf9f7e1474369cb236c20e663a17bcbcae49a3ed/6/Dockerfile#L53)